docs/dapps/public-services.md

```diff
@@ -33,10 +33,10 @@ Mind that the faucet is throttling requests for few minutes.
 ## Node and Explorer APIs
 
 Currently, the following API services are maintained. Note that all APIs are rate limited to prevent spam.
-* `https://wallet-v16.mainnet.alephium.org` for mainnet with node v1.6.X ([Test](https://wallet-v16.mainnet.alephium.org/infos/version))
-* `https://wallet-v16.testnet.alephium.org` for testnet with node v1.6.X ([Test](https://wallet-v16.testnet.alephium.org/infos/version))
-* `https://backend-v112.mainnet.alephium.org` for mainnet with explorer backend v1.12.X ([Test](https://backend-v112.mainnet.alephium.org/infos))
-* `https://backend-v112.testnet.alephium.org` for testnet with explorer backend v1.12.X ([Test](https://backend-v112.testnet.alephium.org/infos))
+* `https://wallet-v17.mainnet.alephium.org` for mainnet with node v1.7.X ([Test](https://wallet-v17.mainnet.alephium.org/infos/version))
+* `https://wallet-v17.testnet.alephium.org` for testnet with node v1.7.X ([Test](https://wallet-v17.testnet.alephium.org/infos/version))
+* `https://backend-v113.mainnet.alephium.org` for mainnet with explorer backend v1.13.X ([Test](https://backend-v113.mainnet.alephium.org/infos))
+* `https://backend-v113.testnet.alephium.org` for testnet with explorer backend v1.13.X ([Test](https://backend-v113.testnet.alephium.org/infos))
```

Comment:
Since these values are being changed, shall we mention somewhere that the following endpoints (current desktop wallet latest release - 1.5.2) are maintained?

- https://wallet-v16.mainnet.alephium.org
- https://backend-v112.mainnet.alephium.org
- https://wallet-v16.testnet.alephium.org
- https://backend-v112.testnet.alephium.org

source: https://github.com/alephium/desktop-wallet/blob/v1.5.2/src/utils/settings.ts#L47-L58

Reply:
If the endpoints I mentioned are considered deprecated, we need to release another desktop wallet version that updates the default network configuration to use the new endpoints (v17, v13)

Reply:
Secondly, `https://backend-v113.testnet.alephium.org` is not accessible yet:
```
NET::ERR_CERT_AUTHORITY_INVALID
```

Reply:
I corrected all your comments above, thanks for 👀